saleor/plugins/sendgrid/plugin.py

```diff
@@ -213,10 +214,17 @@ def notify(self, event: NotifyEventType, payload: dict, previous_value):
         if not self.active:
             return previous_value
 
-        if event not in UserNotifyEvent.CHOICES:
+        event_in_notify_event = event in UserNotifyEvent.CHOICES
+        event_in_event_map = event in EVENT_MAP
+
+        if not event_in_notify_event or not event_in_event_map:
```

Comment:
I would say, that we shouldn't trigger a dynamic template for the events defined by Saleor but not defined in SendgridPlugin.
IMHO It should be:
`if not event_in_notify_event:`  instead of `if not event_in_notify_event or not event_in_event_map:`